### PR TITLE
Add policy for ipc_t

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by dougsland** [comment]
CI/CD error not related to the patch: [02:32:34] [E] [test-schedule-runner] Exception raised: Guest couldn't be provisioned: Artemis resource ended in 'error' state

**Comment by dougsland** [comment]
LGTM waiting feedback.

File: qm.te [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+role system_r types ipc_t;
+unconfined_domain_noaudit(ipc_t)
+
+type ipc_exec_t;
+application_executable_file(ipc_exec_t)
+allow ipc_t { ipc_exec_t container_file_type}:file entrypoint;
+
+init_system_domain(ipc_t, ipc_exec_t)
+role system_r types ipc_t;
+domtrans_pattern(container_runtime_domain, ipc_exec_t, ipc_t)
```
Comment (Yarboa) [inline qm.te:19]: @rhatdan as per this logs of rpm build 
https://download.copr.fedorainfracloud.org/results/packit/containers-qm-454/fedora-40-x86_64/07612266-qm/builder-live.log.gz

You missed the `;`

Comment (Yarboa) [inline qm.te:19]:
```suggestion
domtrans_pattern(container_runtime_domain, ipc_exec_t, ipc_t);
```

File: qm.te [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+role system_r types ipc_t;
+unconfined_domain_noaudit(ipc_t)
+
+type ipc_exec_t;
+application_executable_file(ipc_exec_t)
+allow ipc_t { ipc_exec_t container_file_type}:file entrypoint;
+
+init_system_domain(ipc_t, ipc_exec_t)
+role system_r types ipc_t;
+domtrans_pattern(container_runtime_domain, ipc_exec_t, ipc_t);
```
Comment (Yarboa) [inline qm.te:19]:
```suggestion
domtrans_pattern(container_runtime_domain, ipc_exec_t, ipc_t)
```

**Review by Yarboa — COMMENTED** [review]
rpm build is failing, tried to fix it :/ but i realize that i will have to test it locally
Please see rpm build status

File: qm.te (hunk 1 above)
Comment (Yarboa) [inline qm.te:19]: Is this defined somewhere? 
`container_runtime_domain` it seems that rpm build is failing on it
```
qm.te:19:ERROR 'unknown type container_runtime_domain'
```

**Comment by rhatdan** [comment]
Once this passes CI, lets merge and then we can add minor changes to ipc policy.

**Review by Yarboa — APPROVED** [review]

**Pull request merged**